ktor-http/common/src/io/ktor/http/Parameters.kt

```diff
@@ -75,6 +88,14 @@ fun parametersOf(vararg pairs: Pair<String, List<String>>): Parameters = Paramet
 @Suppress("KDocMissingDocumentation")
 @InternalAPI
 class ParametersImpl(values: Map<String, List<String>> = emptyMap()) : Parameters, StringValuesImpl(true, values) {
+    private var option: UrlEncodingOption = super.urlEncodingOption
```

Comment:
Looks like a hack for me.

Move `urlEncodingOption` to primary constructor and provide a secondary constructor to keep abi (the same applies to `ParametersBuilder` too).

All other things LGFM

Reply:
Now the code looks straightforward to me 🙂 Thanks a lot for many useful advice!